x-pack/packages/kbn-ecs-data-quality-dashboard/impl/data_quality/data_quality_panel/remote_clusters_callout/translations.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const TITLE = i18n.translate('ecsDataQualityDashboard.remoteClustersCallout.title', {
+  defaultMessage: "Remote clusters won't be checked",
+});
+
+export const TO_CHECK_INDICES_ON_REMOTE_CLUSTERS = i18n.translate(
+  'ecsDataQualityDashboard.remoteClustersCallout.toCheckIndicesOnRemoteClustersLabel',
+  {
+    defaultMessage:
+      "To check indices on remote clusters supporting cross-cluster search, login to the remote cluster's Kibana",
```

Comment:
SUPER nit 😂, but in this context I think login would be two separate words 'log in'

Reply:
😂 This is the nittiest of nits, please feel free to ignore, but I'll approve if you do end up doing it 😹